app/assets/javascripts/discourse/app/components/topic-navigation.js

```diff
@@ -9,11 +9,11 @@ import discourseDebounce from "discourse-common/lib/debounce";
 import { headerOffset } from "discourse/lib/offset-calculator";
 import { next } from "@ember/runloop";
 import discourseLater from "discourse-common/lib/later";
-import { observes } from "discourse-common/utils/decorators";
+import { bind, observes } from "discourse-common/utils/decorators";
 import JumpToPost from "./modal/jump-to-post";
 
-const MIN_WIDTH_TIMELINE = 924,
-  MIN_HEIGHT_TIMELINE = 325;
+const MIN_WIDTH_TIMELINE = 925;
```

Comment:
In css there's progress-bar's `(max-width: 924px)` and timeline's `(min-width: 925px)` so now it matches it in the name and value